bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/AbstractMyStromHandler.java

```diff
@@ -147,8 +147,8 @@ private void initializeInternal() {
         try {
             updateProperties();
             checkRequiredInfo();
-            updateStatus(ThingStatus.ONLINE);
             pollingJob = scheduler.scheduleWithFixedDelay(this::pollDevice, 0, config.getRefresh(), TimeUnit.SECONDS);
+            updateStatus(ThingStatus.ONLINE);
```

Comment:
Why was this moved? To be honest, it seems wrong either way. `initialize()` is correctly setting status **UNKNOWN** before scheduling this method. Each handler implementation seems to set **ONLINE** in `pollDevice` after actually receiving data. Previously this method would already have already have set **ONLINE**, and now it can happen at the same time here and within the concrete handler implementation.

I would suggest to remove this status update, if you can verify that all handlers will set status **ONLINE** in `pollDevice` and test that at least one of the handlers will provide correct transitions from UNINITIALIZED -> INITIALIZING -> UNKNOWN -> ONLINE.

Reply:
You're right, I removed the function call and it now works like charm.